lib/topologies/shared.js

```diff
@@ -378,11 +378,6 @@ function resolveClusterTime(topology, $clusterTime) {
 //       to share code.
 const SessionMixins = {
   endSessions: function(sessions, callback) {
-    if (this.isConnected()) {
```

Comment:
this is insane, I can't believe I committed it 😄 It _must_ have been accidentally inverted? `if (!this.isConnected()) { // bail }`?